source/network/palo-alto/palo-failover.html.md.erb

```diff
@@ -0,0 +1,92 @@
+---
+title: Palo Alto Failover
+weight: 10
+last_reviewed_on: 2025-01-10
+review_in: 4 months
+---
+# <%= current_page.data.title %>
+## Overview
+
+For applications accessed via Cloud Gateway, we have pinned routes in place to route traffic through
+a specific Palo Alto firewall, to ensure the return traffic is routed back through the same firewall.
+
+In production, by default these pinned routes target 10.11.8.37 which is [hmcts-hub-prod-int-palo-vm-1](https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/0978315c-75fe-4ada-9d11-1eb5e0e0b214/resourceGroups/hmcts-hub-prod-int/providers/Microsoft.Compute/virtualMachines/hmcts-hub-prod-int-palo-vm-1/overview)
+and should be updated to 10.11.8.38 which is [hmcts-hub-prod-int-palo-vm-0](https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/0978315c-75fe-4ada-9d11-1eb5e0e0b214/resourceGroups/hmcts-hub-prod-int/providers/Microsoft.Compute/virtualMachines/hmcts-hub-prod-int-palo-vm-0/overview) in the event of a failover is needed in production.
+
+This means that in the event of palo-vm-1 being unavailable, either due to an incident, or scheduled maintenance, traffic from Cloud Gateway will not be able to reach the applications.
+
+Applications affected by this are:
+- DARTs, LibraGoB, SDP, SDP, Juror Digital, CVP, BAIS, MI, Interim Hosting
+
+This document will describe the process of failing over to the other firewall in the pair. 
+
+
+## Failover Process
+Before starting the failover process, be aware that there may be a brief period of downtime while the failover is in progress.
+
+Where possible, James Drew (James.Drew2@justice.gov.uk) and Kalyan Deevanapalli (kalyan.deevanapalli@hmcts.net) should be contacted in advance of the failover. 
+
+There are three places across two repositories where the IP address needs to be updated:
+[aks-sds-deploy](https://github.com/hmcts/aks-sds-deploy) and [azure-platform-virtualwan](https://github.com/hmcts/azure-platform-virtualwan)
+
+1. Update the IP address in the following files in the aks-sds-deploy repository:
+
+Pinned aks routes in [prod-pinned-aks-routes.yaml](https://github.com/hmcts/aks-sds-deploy/blob/master/environments/01-network/prod-pinned-aks-routes.yaml#L3) and
+Pinned app gateway routes [prod-pinned-appgw-routes.yaml](https://github.com/hmcts/aks-sds-deploy/blob/master/environments/01-network/prod-pinned-appgw-routes.yaml#L3)
+
+See example [PR for DEMO enviornment](https://github.com/hmcts/aks-sds-deploy/pull/662/files)
+**Note:** the IPs are different for DEMO and PROD environments.
+
+2. Update the IP address in the following files in the azure-platform-virtualwan repository:
+
+static vnet routes in [prod.tfvars](https://github.com/hmcts/azure-platform-virtualwan/blob/b48a0cc40b52accfd6884b8811d2e79503d071be/environments/prod/prod.tfvars#L153-L156)
```

Comment:
I am unsure of just referencing line numbers - what happens that changes? Maybe add exampl of changes that you want to change?